web_service/xmlrpc_test.py

```diff
@@ -0,0 +1,32 @@
+import xmlrpc.client
+
+server = "localhost"
+port = 8069
+db_name = "practice"
+user = "admin"
+password = "admin"
+api_key="c4c17454847c978f7ed8cc2a400c9b6eda3cd6ea"
+args = {
+    'name':'sesion desde api',
+    'number_seats':'111',
+    'course_id':'2', 
+}
+
+root = 'http://%s:%d/xmlrpc/' % (server, port)
+uid = xmlrpc.client.ServerProxy(root + 'common').login(db_name, user, password )
+sock = xmlrpc.client.ServerProxy(root + 'object')
+
+
+
+list_session_before = sock.execute(db_name,2,password,"open.academy.session","search_read",[])
```

Comment:
1. Do [this](https://github.com/edy1192/openacademy-project/blob/master/web_services/ws_test.py#L15-L17) to avoid call execute method in every sentence. 
2. Do not use hard coded user id please.
3. Search only the fields you need (`name, number_seats, course_id`).